nomic/chain.json

```diff
@@ -36,27 +42,27 @@
   },
   "codebase": {
     "git_repo": "https://github.com/nomic-io/nomic",
-    "recommended_version": "v8.0.0",
+    "recommended_version": "v9.0.0",
     "compatible_versions": [
-      "v8.0.0"
+      "v9.0.0"
     ],
     "consensus": {
       "type": "tendermint",
-      "version": "v0.32.0"
+      "version": "v0.34.0"
     },
     "genesis": {
       "genesis_url": "https://raw.githubusercontent.com/nomic-io/nomic/develop/genesis/stakenet-3.json"
     },
     "versions": [
       {
-        "name": "v8.0.0",
```

Comment:
Please retain v8 within the version history. v9 would be added as a new object below.